requirements/verification.txt

```diff
@@ -11,3 +11,5 @@ statsmodels
 seaborn
 tqdm
 argparse
+daiquiri
+multiprocessing
```

Comment:
multiprocessing is in the standard library so we don't need it in the requirements.